### DISCUSSION TITLE
Inference cost of Microsoft's PodcastCopilot?

### DISCUSSION THREAD

**Discussion by tigco** [taishi-i/awesome-ChatGPT-repositories#30] [Q&A]
Has anyone deployed https://github.com/microsoft/PodcastCopilot? I wonder how much each run costs.

**Comment by taishi-i** [comment]
Hi @tigco! What is the cost associated with running this repository? Is it for OpenAI's API or something else?

**Reply by tigco** [reply]
Hi! I'm referring to the Microsoft's Podcast Copilot repository (https://github.com/microsoft/PodcastCopilot). Has anyone deployed it? I'm curious to know the approximate cost to process a single podcast episode using it.
